components/theme.ts

```diff
@@ -47,10 +47,10 @@ export const theme = extendTheme({
     Text: {
       variants: {
         clickable: {
+          textDecoration: 'underline',
           cursor: 'pointer',
           _hover: {
             fontWeight: 'bold',
-            textDecoration: 'underline',
```

Comment:
Think it makes sense to have clickable text always be underlined, not just on hover

Reply:
I have no preference here